### ISSUE TITLE
proxy: unknown scheme: http

### ISSUE THREAD

**Issue by vectorchen** [headzoo/surf#89]
I setted up a proxy server in local PC with nginx.
But when my configuration code as follow:
`err = bow.SetProxy("http://192.168.1.120:82")
	if err != nil {
		fmt.Println(err)
	}`
when I run my pro, then it show me as the title!
Please help, thanks in advance！

**Comment by vectorchen** [comment]
![image](https://user-images.githubusercontent.com/14350150/34862814-443ae446-f7a7-11e7-96ca-20ab37e4ff2e.png)
![image](https://user-images.githubusercontent.com/14350150/34862825-50a5d254-f7a7-11e7-9ed8-6c2c4e890631.png)

**Comment by Joffcom** [comment]
Have you tried setting the proxy using the SetTransport method like you would with the http client? Should act as a good work around for you.

**Comment by vectorchen** [comment]
@Joffcom I looked up source code , It contains the method setTransport, as follow:
![image](https://user-images.githubusercontent.com/14350150/34924704-e9319bc2-f9df-11e7-8dfb-d4ba85ef8ef6.png)

**Comment by Joffcom** [comment]
I would still give it a Bash but instead of dial use the Proxy option to see if it works.

You could try setting an HTTP_PROXY envvar to see if that sorts it as well

**Comment by vectorchen** [comment]
@Joffcom Yes, I've already tried, but it doesn't work. Meanwhile I need to implement IP switching dynamicly.

**Comment by vectorchen** [comment]
I've  got an answer from myself .proxy address must be in this pattern: "http://192.168.1.120:82/"

**Comment by Joffcom** [comment]
Nice work so all it needed was the trailing the /
